hooks/post/30-upgrade.rb

```diff
@@ -1,3 +1,7 @@
+require 'fileutils'
+
+STEP_DIRECTORY='/etc/foreman-installer/hooks/post/'
```

Comment:
That's fine then, just making it more obvious what it is should be good enough.  e.g.,  `applied_hooks` like we talked about on IRC